HM3CoreKill/HM3CoreKill/sdk/ZStandardMenuFactory.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstdint>
+
+namespace ioi { namespace hm3 {
+	class ZStandardMenuFactory
+	{
+	public:
+		virtual void Function_0000(); //#0 +0 .rdata:00780f6c
+		virtual void Function_0001(); //#1 +4 .rdata:00780f70
+		virtual void Function_0002(); //#2 +8 .rdata:00780f74
+		virtual std::uintptr_t getMenuElementByName(const char* elementClassName); //#3 +c .rdata:00780f78
```

Comment:
I guess we should return ZGUIBase* instead abstract std::uintptr_t (we need to check that this method returns actually pointer to this or all returned pointers are derived from ZGUIBase class)